doc/source/reference/routines.ma.rst

```diff
@@ -107,6 +109,7 @@ Changing the shape
    ma.MaskedArray.ravel
    ma.MaskedArray.reshape
    ma.MaskedArray.resize
+   ma.diagflat
```

Comment:
According to the definition, I think this function belongs in the `Creation > From existing data section`.

Reply:
It also changes the dimension of the array